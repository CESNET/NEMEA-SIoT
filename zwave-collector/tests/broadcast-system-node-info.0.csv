time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2019-07-29T17:38:45.935,1,fe90821d03010522ff0101d39c0110015e22855970565a7a72328e717398312586ebeb
