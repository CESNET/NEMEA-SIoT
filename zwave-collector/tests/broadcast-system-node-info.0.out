3,2019-07-29T17:38:45.935,4270883357,0,1,0,255,0,0,5,34,0,"System","Node Info",,0101d39c0110015e22855970565a7a72328e717398312586,"","Broadcast"
